### PR TITLE
Fix memory sizes for Spark Nodes

### PR DESCRIPTION
**Pull request by mwc360**
These sizes no longer align with what the UI shows

### CODE REVIEW

**Comment by Jak-MS** [comment]
@santhoshravindran7 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by SnehaGunda** [comment]
@mwc360 Can you please share a screenshot of where you see the values?

**Comment by mwc360** [comment]
> @mwc360 Can you please share a screenshot of where you see the values?

Yes, the only place in Fabric that the # of cores and RAM is visible is when creating/editing Environments. I updated the values in the docs based on the memory value that the UI shows in Environments for each node size (I also confirmed in the spark configs when using the respective spark pool).

image

**Comment by santhoshravindran7** [comment]
sign-off

**Comment by santhoshravindran7** [comment]
#sign-off

**Review by santhoshravindran7 — APPROVED** [review]

**Comment by santhoshravindran7** [comment]
#sign-off

File: docs/data-engineering/spark-compute.md [hunk 1]
```diff
@@ -69,15 +69,15 @@ An Apache Spark pool instance consists of one head node and worker nodes, could
 
 ## Node sizes
 
-A Spark pool can be defined with node sizes that range from a small compute node (with 4 vCore and 32 GB of memory) to a large compute node (with 64 vCore and 512 GB of memory per node). Node sizes can be altered after pool creation, although the active session would have to be restarted.
+A Spark pool can be defined with node sizes that range from a small compute node (with 4 vCore and 28 GB of memory) to an double extra large compute node (with 64 vCore and 400 GB of memory per node). Node sizes can be altered after pool creation, although the active session would have to be restarted.
```
Comment (Court72) [inline docs/data-engineering/spark-compute.md:72]:
```suggestion
A Spark pool can be defined with node sizes that range from a small compute node (with 4 vCore and 28 GB of memory) to a double extra large compute node (with 64 vCore and 400 GB of memory per node). Node sizes can be altered after pool creation, although the active session would have to be restarted.
```

**Pull request merged**